File: Expressions & Assignment/NullishCoalescing.py
username = None
display_name = username if username is not None else "Guest"

user = {"profile": {"bio": "Hello!"}}
profile = user.get("profile")
bio = profile.get("bio") if profile else None
address = user.get("address")
missing = address.get("city") if address else None

print(display_name)
print(bio)
print(missing)

File: Expressions & Assignment/NullishCoalescing.js
let username = null;
let displayName = username ?? "Guest";   // "Guest" — ?? checks null/undefined only

let user = { profile: { bio: "Hello!" } };
let bio = user?.profile?.bio;            // "Hello!"
let missing = user?.address?.city;       // undefined — no error thrown

console.log(displayName); // "Guest"
console.log(bio);         // "Hello!"
console.log(missing);     // undefined